molo/yourtips/models.py

```diff
@@ -71,6 +73,11 @@ class YourTip(TranslatablePageMixinNotRoutable, Page):
             "Styling options that can be applied to this section "
             "and all its descendants"))
 
+    homepage_action_copy = models.CharField(
+        default='Do you have advice '
+                'you can share with other youth on relationships?',
```

Comment:
Can we add the default text in the template instead, this text needs to be in a trans block so we can get the translation for it.